src/api/entities/Identity/Portfolios.ts

```diff
@@ -84,4 +84,14 @@ export class Portfolios extends Namespace<Identity> {
   public createPortfolio(args: { name: string }): Promise<TransactionQueue<NumberedPortfolio>> {
     return createPortfolio.prepare(args, this.context);
   }
+
+  /**
+   * Delete a Portfolio by ID
+   */
+  public deletePortfolio(args: { portfolioId: BigNumber }): Promise<TransactionQueue<void>> {
```

Comment:
Just `delete`

Also please change the `createPortfolio` endpoint above to `create` (and list it as a breaking change)

Reply:
Args should be `{ portfolio: BigNumber | Portfolio }`